### PR TITLE
Dennis/menu sharing updated

### PR DESCRIPTION
**Pull request by dennisfedorko**

### CODE REVIEW

File: Classes/Eat Now/MenuImages.swift [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+class MenuImages: NSObject {
+
+    // returns an image that contains the menu categories + items
+    // looks like table view of menu
+    // takes in a menuIterable list which can be generated from the event class or eatery class
+    class func createMenuItemsImage(width:CGFloat, menuIterable:[(String,[String])]) -> UIImage {
+        let separatorColor = UIColor.whiteColor()
+        let bodyColor = UIColor(red: 245/255.0, green: 245/255.0, blue: 245/255.0, alpha: 1.0)
+        let categoryHeaderHeight:CGFloat = 48
+        let dividerSize:CGFloat = 20
```
Comment (lucasderraugh) [inline Classes/Eat Now/MenuImages.swift]: I'm annoying, but just move your types to be `name: CGFloat` (space before type)

File: Classes/Eat Now/MenuImages.swift [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+        //add all components to completeMenuView
+        var y:CGFloat = 0
+        for categoryView in categoryViews {
+            //set y for category view
+            var frame = categoryView.frame
+            frame.origin.y = y
+            categoryView.frame = frame
+            
+            completeMenuView.addSubview(categoryView)
+            y = y + categoryView.frame.height
```
Comment (lucasderraugh) [inline Classes/Eat Now/MenuImages.swift]: `+=` please

**Comment by lucasderraugh** [comment]
Overall, please just change all the places in your code with `name: type` where there is a space after the colon. Also, if you want to remove the text for the menu sharing I'd be ok with that.

File: Classes/Eat Now/MenuImages.swift (hunk 1 above)
Comment (dennisfedorko) [inline Classes/Eat Now/MenuImages.swift]: f56edde

File: Classes/Eat Now/MenuImages.swift (hunk 2 above)
Comment (dennisfedorko) [inline Classes/Eat Now/MenuImages.swift]: 2939577

**Comment by dennisfedorko** [comment]
4c33d1d removed share sheet text

**Comment by dennisfedorko** [comment]
merged 5b0407bf5d72279e9d470eff3f5c18590dd11951

**Pull request closed without merging**